packages/core/src/sync-historical/service.ts

```diff
@@ -952,37 +953,40 @@ export class HistoricalSyncService extends Emittery<HistoricalSyncEvents> {
     fromBlock: Hex;
     toBlock: Hex;
   }): Promise<RpcLog[]> => {
+    const _params: ParseGetLogsErrorParameters["params"] = [
```

Comment:
I feel like there's probably a Viem type that's better here? Maybe not though.

Reply:
Couldn't find one